README.md

```diff
@@ -35,6 +35,32 @@ Jan Roehrich ([GitHub](https://github.com/roehrijn))
 
 Please find the documentation on the [Jenkins Wiki page Amazon EC2 Container Service Plugin](https://wiki.jenkins-ci.org/display/JENKINS/Amazon+EC2+Container+Service+Plugin).
 
+## Declarative Pipeline
+Declarative Pipeline support requires Jenkins 2.66+  
+
+Declarative agents can be defined like shown below. You can also reuse pre-configured templates and override certain settings using `inheritFrom`.  
+
+```groovy
+pipeline {
+  agent none
+
+  stages {
+    stage('Test') {
+        agent {
+            ecs {
+                inheritFrom 'my-preconfigured-template'
+                cpu 2048
```

Comment:
probably the wrong place to comment, but do we have a way to limit the maximum parameter for CPU and memory, so that people can't set arbitrarily high values and break the cluster / bring it to an infinite scaling loop

Reply:
done with https://github.com/jenkinsci/amazon-ecs-plugin/pull/79/commits/0d708b6ac94465b8d68b3cf27811e6ce72d17357